examples/main_direct.py

```diff
@@ -1,20 +1,22 @@
 """
 Example script that calls the main function directly
 
-Note that this needs to be moved to the main directory prior to running since Python does not seem to enjoy
- parallel directories.
-Also, create data in the tests directory per the structure below prior to running.
+From the cloned rosbag-merge directory :
+
+  python3 -m pip install .
+  python3 examples/main_direct.py
+  
 """
 
-from src.rosbag_merge.main import main
+from rosbag_merge.main import main
```

Comment:
Uses the assumed installed package which a developer would have installed via `python3 -m pip install .` .

Reply:
Looks good. This assumption makes it easier.